Automattic/wp-calypso
# Fix vertical alignment of notice component

Closes #75270

## Proposed Changes

This PR adds a negative margin to the content of the Notice component.

| Before | After |
|--------|------|
| ![CleanShot 2023-04-04 at 14 20 28@2x](https://user-images.githubusercontent.com/528287/229789858-e09a9fca-eefd-4025-8793-9bffbae6109f.png) | ![CleanShot 2023-04-04 at 14 21 14@2x](https://user-images.githubusercontent.com/528287/229789909-b1008f0a-edad-4d51-a023-e97ea1461ba1.png) |

There was a bug in the previous implementation, it was using CSS variables that were unavailable:

<img width="150" alt="CleanShot 2023-04-04 at 13 42 49@2x" src="https://user-images.githubusercontent.com/528287/229790223-5658905d-48df-454f-b5d3-01e7f4876efc.png">

That's been fixed as well, by using the SCSS variables. I replaced `font-body-small` with `font-body`, as that was the size that was used before, and `font-body-small` didn't look right.

## Testing Instructions

1. Apply this PR and make sure you have a cookie subkey set
2. Navigate to http://calypso.localhost:3000/subscriptions/sites
3. Delete all your subscriptions
4. Verify 

## Pre-merge Checklist
